### DISCUSSION TITLE
Translation impossible on Transiflex

### DISCUSSION THREAD

**Discussion by VertOurs** [AppOutlet/AppOutlet#259] [Q&A] [answered]
**Describe the bug**
 I want to contribute to the translation of the application into French. On the Transiflex page, despite the presence of French in the project languages, it is not possible to contribute.

**To Reproduce**
1. Go to the web page ' https://www.transifex.com/app-outlet/app-outlet/ ' with a Transiflex account.
2. Scroll down to the bottom of the page and click on the 'show all languages' button to see that French is one of the 14 languages available for translation.
3. At the top of the page on the right, click on the blue button 'join the team' and search for French in the languages offered. There are only 13 languages available and not 14, because French is not present in the list of the form.

**Expected behavior**
I would like the French language to be available in this menu in order to translate the application

**Comment by MessiasLima** [comment, marked as answer]
I have already accepted your join request. Could you please verify if you are already able to translate?

**Reply by VertOurs** [reply]
Thank you for your reactivity. Indeed, I am able to translate now. I just finished translating 100%. Without context, some translations may be too literal, I will correct them if needed when I see them in context.

**Reply by MessiasLima** [reply]
don't worry, I will take a look and let you know if I find something that could be improved. thank you a lot for your help
